### PR TITLE
chore: remove napi exclusions

### PR DESCRIPTION
**Pull request by williamluke4**
- [x]  aws-graviton
- [ ]  ~~azure-functions-linux~~ (still excluded)
- [ ]  ~~studio~~  (still excluded)
- [ ]  ~~azure-functions-windows~~ (still excluded)
- [x]  serverless-framework-lambda
- [ ]  ~~node: 10.16.0~~ (not supported by `napi-rs`)
- [x]  node: 12.2.0

Remaining exclusions moved to this [issue](https://github.com/prisma/e2e-tests/issues/1748)

### CODE REVIEW

File: platforms-serverless/serverless-framework-lambda/test.ts [hunk 1]
```diff
@@ -9,11 +9,13 @@ async function main() {
   console.log({ data: data.$response.data })
 
   const actual = (data.$response.data as any).Payload
-  const binaryString = `,"files":["index-browser.js","index.d.ts","index.js","package.json","query-engine-rhel-openssl-1.0.x","schema.prisma"]`
+  const binaryString = process.env.PRISMA_FORCE_NAPI === 'true'
+    ? `,"files":["index-browser.js","index.d.ts","index.js","libquery_engine_napi-debian-openssl-1.1.x.so.node","libquery_engine_napi-rhel-openssl-1.0.x.so.node","package.json","schema.prisma"]`
```
Comment (janpio) [inline platforms-serverless/serverless-framework-lambda/test.ts]: You can get to this:
```suggestion
    ? `,"files":["index-browser.js","index.d.ts","index.js","libquery_engine_napi-rhel-openssl-1.0.x.so.node","package.json","schema.prisma"]`
```
by adding this to `serverless.yml` in this project at the bottom:
```
    - '!node_modules/.prisma/client/libquery_engine_napi-*'
    - 'node_modules/.prisma/client/libquery_engine_napi-rhel-openssl-1.0.x.so.node'
```

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -225,12 +222,10 @@ jobs:
           - 14.0.0 # minimal minor version of node 14 via https://www.prisma.io/docs/reference/system-requirements
           - 15
           - 16
-        os: [ubuntu-latest] #, windows-latest, macos-latest]    
+        os: [ubuntu-latest] #, windows-latest, macos-latest]
         exclude:
         - engine: napi
           node: 10.16.0
```
Comment (janpio) [inline .github/workflows/test.yaml]: TODO or comment

File: .github/workflows/test.yaml [hunk 3]
```diff
@@ -534,13 +518,101 @@ jobs:
...
+          - netlify-cli
+          - netlify-ci
+          - gcp-functions
+          - firebase-functions
+          - azure-functions-linux
+          - azure-functions-windows
+          - serverless-framework-lambda
+        os: [ubuntu-latest] #, windows-latest, macos-latest]
+        exclude:
+        - platform: azure-functions-windows
```
Comment (janpio) [inline .github/workflows/test.yaml]:
```suggestion
        - platform: azure-functions-windows # 32 bit Node by default, see TODO issue link
```

**Review by janpio — APPROVED** [review]
Beautiful if tests pass

File: .github/workflows/test.yaml [hunk 4]
```diff
@@ -534,13 +570,102 @@ jobs:
...
+          timeout_minutes: 60
+          max_attempts: 3
+          command: bash .github/scripts/test-project.sh platforms-serverless ${{ matrix.platform }}
+
+      - name: notify-slack
+        if: failure()
+        run: bash .github/slack/notify-failure.sh platforms-serverless ${{ matrix.platform }}
+
+  platforms-serverless-napi:
+    needs: [start-time, platforms-serverless]
```
Comment (janpio) [inline .github/workflows/test.yaml]:
```suggestion
    needs: [start-time, platforms-serverless] # Makes these tests run after their binary counterparts
```

File: .github/workflows/test.yaml [hunk 5]
```diff
@@ -425,13 +419,63 @@ jobs:
...
+          timeout_minutes: 60
+          max_attempts: 3
+          command: bash .github/scripts/test-project.sh platforms ${{ matrix.platform }}
+
+      - name: notify-slack
+        if: failure()
+        run: bash .github/slack/notify-failure.sh platforms ${{ matrix.platform }}
+
+  platforms-napi:
+    needs: [start-time, platforms-napi]
```
Comment (janpio) [inline .github/workflows/test.yaml]:
```suggestion
    needs: [start-time, platforms-napi] # Makes these tests run after their binary counterparts
```

Comment (williamluke4) [reply .github/workflows/test.yaml]: Just realized that it should be `platforms` not `platforms-napi`

Comment (williamluke4) [reply .github/workflows/test.yaml]: Added

File: .github/workflows/test.yaml (hunk 4 above)
Comment (williamluke4) [reply .github/workflows/test.yaml]: Added

File: .github/workflows/test.yaml [hunk 6]
```diff
@@ -406,15 +404,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
```
Comment (janpio) [inline .github/workflows/test.yaml:406]:
```suggestion
      matrix:
        engine: ['binary'] # see extracted napi below
```

File: .github/workflows/test.yaml [hunk 7]
```diff
@@ -467,7 +511,6 @@ jobs:
...
```
Comment (janpio) [inline .github/workflows/test.yaml:514]:
```suggestion
      matrix:
        engine: ['binary'] # see extracted napi below
```

**Pull request merged**